src/System.Collections/tests/Generic/Dictionary/Dictionary.Generic.Tests.netcoreapp.cs

```diff
@@ -161,5 +162,181 @@ public void EnsureCapacity_SetsToNextPrimeNumber()
         }
 
         #endregion
+
+        #region TrimExcess
+
+        [Fact]
+        public void TrimExcess_NegativeCapacity_Throw()
+        {
+            var dictionary = new Dictionary<int, int>();
+            Assert.Throws<ArgumentOutOfRangeException>(() => dictionary.TrimExcess(-1));
```

Comment:
Can we include parameter names with such argument exception validation?  That'll also mean switching these to use AssertExtensions.Throws.